hamcrest-library/src/test/java/org/hamcrest/number/IsCloseToTest.java

```diff
@@ -6,9 +6,10 @@
 import static org.hamcrest.number.IsCloseTo.closeTo;
 
 public class IsCloseToTest extends AbstractMatcherTest {
```

Comment:
@StarBax89 this file now lives at `./hamcrest/src/test/java/org/hamcrest/number/IsCloseToTest.java`

if you rebase from the latest master, it should resolve the conflicts and I can merge in, e.g. `git fetch origin && git rebase origin/master`